cog/daily_charge.py

```
@@ -23,95 +27,243 @@ def get_channels():  # 取得特殊用途頻道的清單，這裡會用來判斷
 
 class Charge(commands.Cog):
-    def __init__(self, bot):
+    def __init__(self, bot: discord.Bot):
         self.bot = bot
-        self.embed = None
 
-    async def send_message(self, point, combo, interaction):
+    async def restore_downtime_point(self):
+        downtime_list = get_downtime_list()
+        unprocessed_downtime_list = list(
+            filter(lambda x: not x.is_restored, downtime_list)
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)